### PR TITLE
SB14-019 Fix expr_eval on character deriving standard character

### PR DESCRIPTION
**Pull request by danielmercier**

### CODE REVIEW

File: user_manual/changes/SB14-019.yaml [hunk 1]
```diff
@@ -0,0 +1,8 @@
+type: bugfix
+title: Fix static evaluator on character literal
+
+description: |
+   The static evaluator were wrongly evaluating a character literal whose
+   type resolves to a type deriving the standard character type.
+
+date: 2019-09-12
```
Comment (Roldak) [inline user_manual/changes/SB14-019.yaml]: Wrong date!

Comment (danielmercier) [reply user_manual/changes/SB14-019.yaml]: haha, thx, will update

File: user_manual/changes/SB14-019.yaml [hunk 2]
```diff
@@ -0,0 +1,8 @@
+type: bugfix
+title: Fix static evaluator on character literal
+
+description: |
+   The static evaluator were wrongly evaluating a character literal whose
```
Comment (pmderodat) [inline user_manual/changes/SB14-019.yaml]: was*

File: user_manual/changes/SB14-019.yaml [hunk 3]
```diff
@@ -0,0 +1,8 @@
...
+   type resolves to a type deriving the standard character type.
```
Comment (pmderodat) [inline user_manual/changes/SB14-019.yaml]: The current wording feels twisted to me. What about: a character literal whose type derives `Standard.Character`?

**Pull request merged**